build.yaml

```diff
@@ -387,6 +389,7 @@ steps:
      # batch, benchmark, auth gsa keys
      kubectl -n {{ default_ns.name }} get -o json secret test-gsa-key | jq '{apiVersion, kind, type, data, metadata: {name: "auth-gsa-key"}}' | kubectl -n {{ default_ns.name }} apply -f -
      kubectl -n {{ default_ns.name }} get -o json secret test-gsa-key | jq '{apiVersion, kind, type, data, metadata: {name: "batch-gsa-key"}}' | kubectl -n {{ default_ns.name }} apply -f -
+     kubectl -n {{ default_ns.name }} get -o json secret test-gsa-key | jq '{apiVersion, kind, type, data, metadata: {name: "atgu-gsa-key"}}' | kubectl -n {{ default_ns.name }} apply -f -
```

Comment:
Right. I believe they were auto-generated in the default namespace with the auth code I added which is why I thought we needed it.

Reply:
Yeah.  This isn't urgent, but a few thoughts:

I could manually create the atgu-gsa instead of creating an account.

I could still create the account, but we could have an option not to create the token.  It is convenient to have the gsa automatically managed by the auth code.

Hmm.